core/src/utils/content/index.ts

```diff
@@ -0,0 +1,98 @@
+import { componentOnReady } from '../helpers';
+import { printRequiredElementError } from '../logging';
+
+const ION_CONTENT_ELEMENT_SELECTOR = 'ion-content';
+const ION_CONTENT_CLASS_SELECTOR = '.ion-content-scroll-host';
+/**
+ * Selector used for implementations reliant on `<ion-content>` for scroll event changes.
+ *
+ * Developers should use the `.ion-content-scroll-host` selector to target the element emitting
+ * scroll events. With virtual scroll implementations this will be the host element for
+ * the scroll viewport.
+ */
+const ION_CONTENT_SELECTOR = `${ION_CONTENT_ELEMENT_SELECTOR}, ${ION_CONTENT_CLASS_SELECTOR}`;
+
+const isIonContent = (el: Element) => el && el.tagName.toLowerCase() === ION_CONTENT_ELEMENT_SELECTOR;
```

Comment:
Minor thing, but can we create an `ION_CONTENT_TAG_NAME` const so we do not have to do `toLowerCase` every time?

```js
const ION_CONTENT_TAG_NAME = 'ION-CONTENT';

...

el && el.tagName === ION_CONTENT_TAG_NAME
```